physics/Interstitials/UFS_SCM_NEPTUNE/GFS_suite_interstitial_4.F90

```diff
@@ -250,13 +264,23 @@ subroutine GFS_suite_interstitial_4_run (im, levs, ltaerosol, tracers_total, ntr
                     !> - Update cloud water mixing ratio
                     qc_mp(i,k) = (clw(i,k,2)-save_qc(i,k))
                     !> - Update cloud water number concentration
-                    gq0(i,k,ntlnc) = max(zero, gq0(i,k,ntlnc) + make_DropletNumber(qc_mp(i,k) * rho, nwfa(i,k)*rho) * orho)
+                    if (imp_physics == imp_physics_thompson) then
```

Comment:
Same comment about adding more 'if' statements within the spatial loops. It's pretty inefficient unless compiler optimizers know how to "fix" it. I don't know if they do or not. Sometimes it's unavoidable, but in this case, it probably is.